apps/vs-code-intellisense/src/types/token-category.ts

```diff
@@ -0,0 +1,16 @@
+export type TokenCategory =
+  | 'background-color'
+  | 'border-color'
+  | 'radius'
+  | 'border-width'
+  | 'box-shadow'
+  | 'data-visualization'
+  | 'font'
+  | 'font-size'
+  | 'font-weight'
+  | 'line-height'
+  | 'color'
+  | 'sizing'
+  | 'spacing'
+  | 'text-color'
+  | 'z-index';
```

Comment:
hey, could you explain what is the difference between TokenCategory and TokenType?

Reply:
`type` seems to be the most specific and `category` seems to be the parent grouping. These were pulled from the [`tokens.raw.json`](https://unpkg.com/browse/@twilio-paste/design-tokens@8.5.0/dist/tokens.raw.json) file in the design-tokens package, i.e.

```json
"color-background-user": {
  "type": "color",
  "category": "background-color",
  "value": "#5817bd",
  "comment": "User avatar background color.",
  "originalValue": "{!palette-purple-70}",
  "name": "color-background-user"
}
// ...
"font-size-10": {
  "category": "font-size",
  "type": "font-size",
  "value": "0.625rem",
  "comment": "Constant typography token for font size 10",
  "originalValue": "{!font-size-10}",
  "name": "font-size-10"
}
```

Reply:
I got it, thanks